### PR TITLE
Converting GraphClass.Rnw to GraphClass.Rmd

### PR DESCRIPTION
**Pull request by Chisomgold**
Hi @jwokaty 

I just made the sweave to Rmd conversion for GraphClass as part of the Sweave2Rmd project.

Attached are the files for comparison: [GraphClass PDF file](https://bioconductor.org/packages/devel/bioc/vignettes/graph/inst/doc/GraphClass.pdf) and [GraphClass HTML](https://github.com/Bioconductor/graph/files/9838471/GraphClass.zip)

### CODE REVIEW

**Review by jwokaty — CHANGES_REQUESTED** [review]
Great job! I just have a few more items to add for your review. 

- [x] Can I ask that you also update the `DESCRIPTION` file such that the `Author` section is formatted similar to the [BiocParallel DESCRIPTION file](https://github.com/Bioconductor/BiocParallel/blob/master/DESCRIPTION#L5-L21)? Add yourself as a contributor like the two contributors from line 16.
- [x] Also, replace the `Maintainer` line with
    ```
    person(
        "Bioconductor Package Maintainer",
        role = "cre",
        email = "maintainer@bioconductor.org"
    )
    ```
- [x] Change the section numbering so that they aren't decimals--e.g., "1" rather than "0.1". I think you can do this by removing an extra `#` for the sections. For example, the first section should be `# Introduction`. Then you should adjust the sections that follow.

File: vignettes/GraphClass.Rmd [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+The second specialized class is a *clusterGraph*, which can be used to represent the output of a clustering algorithm as a graph. Samples represent nodes and all samples in the same cluster have edges, while samples in distinct clusters do not. Instances of this class must have their edgemode as `undirected`. If the edgemode is reset, then coercion to some other mode of graph is needed.
+
+### Methods of graphs
+
+Here are some of the methods that all graph-like objects should support:
+
+`nodes(object)` Returns a character vector of the node labels. The order is not defined.
+
+`nodes<-(object)` Returns a new graph object with the node labels set as specified by a character vector. This is slightly fragile since the order matters. But the order can only really be determined by first calling the R function, `nodes`, and then providing a character vector of the appropriate length.
```
[the remark this answers is not in the document]
Comment (Chisomgold) [reply vignettes/GraphClass.Rmd]: Hi @jwokaty 

Thank you for the feedback.

I've made the changes as requested. Here's what the [HTML file looks like now](https://github.com/Bioconductor/graph/files/9859532/GraphClass.zip).

I'll be expecting your remarks

**Review by jwokaty — CHANGES_REQUESTED** [review]
Hi @Chisomgold,

This looks really good, but I need to ask for one edit because my instructions weren't very good.

Can I ask you in the `DESCRIPTION` file remove the `Maintainer` completely and move the value for it to the `Authors@R` vector. It should look like this:

```
Authors@R: c(
    person("R", "Gentleman", role = "aut"),
    person("Elizabeth", "Whalen", role="aut"),
    person("W", "Huber", role="aut"),
    person("S", "Falcon", role="aut"),
    person("Halimat C.", "Atanda", role = "ctb",
        comment = "Converted 'GraphClass' vignette from Sweave to RMarkdown / HTML."
    ),
    person(
        "Bioconductor Package Maintainer",
        role = "cre",
        email = "maintainer@bioconductor.org"
    ))
```

We're trying to develop this convention to make sure that contributors get credit and also use this new style `Authors@R` to include the maintainer. After this edit, I believe we'll be ready to merge your work into Bioconductor!

**Comment by Chisomgold** [comment]
All done @jwokaty

**Comment by jwokaty** [comment]
Great job. Just one comment. Indents in R are typically 4 spaces rather tabs. I went ahead and made the change so we could merge your commit.

**Comment by Chisomgold** [comment]
Thank you. I'll keep it in mind.

**Pull request merged**